### ISSUE TITLE
Use full import path of "appengine" imports

### ISSUE THREAD

**Issue by tbpg** [golang/appengine#146]
Example: https://github.com/golang/appengine/blob/master/internal/api_classic.go#L15

Working on a PR. There are a few spots this shows up.

Found by trying to run `vgo vendor`:

```
$ vgo vendor
vgo: resolving import "appengine/aetest"
vgo: import "appengine/aetest": unknown module appengine/aetest: not a domain name
vgo: resolving import "appengine/cloudsql"
vgo: import "appengine/cloudsql": unknown module appengine/cloudsql: not a domain name
vgo: resolving import "appengine/user"
vgo: import "appengine/user": unknown module appengine/user: not a domain name
vgo: resolving import "appengine_internal"
vgo: import "appengine_internal": unknown module appengine_internal: not a domain name

```

**Comment by broady** [comment]
api_classic *should* import this package, it's the internal package that exists in the GOROOT of the SDK, and in the builder's GOROOT when deployed.

it's correctly behind a build tag.

can you try vgo at HEAD, rather than at 1.0?

**Comment by tbpg** [comment]
Ahh. Interesting. I'll test and report back.

**Comment by tbpg** [comment]
Seems to be fixed by updating `vgo` and the new tag from #145.
